example/v1/buttons/Button.js

```diff
@@ -24,57 +24,76 @@ class Button extends Component {
       onPress,
       buttonStyle,
       clear,
-      loading, loadingStyle, loadingProps,
-      text, textStyle, textProps,
-      icon, iconContainerStyle, iconRight,
+      loading,
+      loadingStyle,
+      loadingProps,
+      text,
+      textStyle,
+      textProps,
+      icon,
+      iconContainerStyle,
+      iconRight,
+      linearGradientProps,
       ...attributes
     } = this.props;
 
     // this is what RN Button does by default
     // https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js#L118
-    const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    const Touchable =
+      Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    const ButtonContainer = linearGradientProps
+      ? require('expo').LinearGradient
```

Comment:
I'd be curious to see what happens when I pass `linearGradientProps` when not using expo. 

Since there is no fallback, I would think this would throw an error. But I am willing to merge it and try it out.